schemas/com.snowplowanalytics.snowplow/media_player_event_end/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event sent when playback stops when end of the media is reached or because no further data is available.",
```

Comment:
Are we able to differentiate between these two states?

Reply:
I am not sure when the end of media happens (I took this description from a previous representation of the end event) and what information is relevant at that point, so not really sure how to differentiate the two states. If you have any suggestions, happy to update this.